GK-js-hw-02/task-04.js

```diff
@@ -0,0 +1,33 @@
+const formatString = function(string) {
```

Comment:
Функцію можна спростити
```suggestion
const formatString = function(string) {
   if (string.length > 40) {
    return `${string.slice(0, 40)}...`;
    }
    return string;
  } 
```